static/thing.js

```diff
@@ -0,0 +1,21 @@
+class Thing {
```

Comment:
Maybe, but it takes too much time for users to name each thing, so perhaps an ID. We'll add it when we need it